tsconfig.json

```diff
@@ -23,10 +23,12 @@
     ],
```

Comment:
this is actually invalid... why should I use the `libs` path ?

please rollback as well

Reply:
The explanation can be found at this article: https://medium.com/@tomastrajan/the-best-way-to-architect-your-angular-libraries-87959301d3d3

```json
"@my-org/some-lib/*": [
  "projects/some-lib/*",
  "projects/some-lib"
],
"@my-org/some-lib": [
  "dist/some-lib/*",
  "dist/some-lib"
]
```

> The **first entry** (with the @my-org/some-lib/* is used during the development when referencing sub-entries from each other. The **second entry** is used during the build to produce library artifacts that can be consumed by the applications in desired way.

So if you planning to add secondary entry points (like Angular does with `@angular/common`, `@angular/core`, etc), this would help to properly reference those entry points.